wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController+MessageAction.swift

```diff
@@ -161,7 +161,8 @@ extension ConversationContentViewController {
             client.resetSession()
         case .react(let reaction):
             userSession.perform {
-                message.react(reaction)
+                let useCase = self.userSession.makeToggleMessageReactionUseCase()
```

Comment:
we could remove the encapsulating userSession.perform and save within the usecase, no?